tests/braille-specs/de-g2.yaml

```diff
@@ -723,3 +723,49 @@ tests:
     - te}...
   - - Test ...
     - te} ...
+
+  # Missing dot 6 before single letters in front of apostrophe
```

Comment:
Maybe better to describe the expected behavior / the rule in the braille book, instead of the bug?

Reply:
Asked Mischa and added a better comment now